contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics/__init__.py

```diff
@@ -24,8 +26,9 @@
     import ProcessMemoryMetric
 
 # List of standard metrics to track
-STANDARD_METRICS = [AvailableMemoryMetric, ProcessCPUMetric,
-                    ProcessMemoryMetric, ProcessorTimeMetric]
+STANDARD_METRICS = [AvailableMemoryMetric, DependencyRateMetric,
+                    ProcessCPUMetric, ProcessMemoryMetric,
```

Comment:
Personally I think it could be better to put each item on its own line, on strong opinion though.

Reply:
I like it better as well. Makes it less annoying to reformat and alphabetize.